docs/scraping-receivers.md

```diff
@@ -0,0 +1,149 @@
+stability level:
+
+- Adding a new optional metric.
+
+Most of other changes to the emitted metrics are considered breaking and MUST be handled according to the stability 
+level of the receiver. Each type of breaking change defines a set of steps that MUST (or SHOULD) be applied across 
+several releases for a Stable (or Beta) components. At least 3 versions SHOULD be kept between the steps to give 
+users time to prepare, e.g. if the first step is released in v0.62.0, the second step SHOULD be released not earlier 
+than 0.65.0. Any warnings SHOULD include the version starting from which the next step will take action. If 
+breaking change is more complicated and many metrics are involved in the change, feature gates SHOULD be used instead.
+
+### Removing an optional metric
+
+Steps to remove an optional metric:
+
+1. Mark the metric as deprecated in `metadata.yaml` by adding "[DEPRECATED]" in its description. Show a warning that 
+   the metric will be removed if the `enabled` option is set explicitly to `true` in user settings. 
+2. Remove the metric.
+
+### Removing a default metric
+
+Steps to remove a default metric:
+
+1. Mark the metric as deprecated in `metadata.yaml` by adding "[DEPRECATED]" in its description. Show a warning that
+   the metric will be removed if the `enabled` option is not explicitly set to `false` in user settings.
+2. Turn the metric into optional. Show a warning that the metric will be removed if the `enabled` option is set to 
+   `true` in user settings.
+3. Remove the metric.
+
+### Making a default metric optional
+
+Steps to turn a metric from default to optional:
+
+1. Add a warning that the metric will be turned into optional if `enabled` field is not set explicitly to any value in
+   user settings. Warning example: "WARNING: Metric `foo.bar` will be disabled by default in v0.65.0. If you want to
+   keep it, please enable it explicitly in the receiver settings."
+2. Remove the warning and update `metadata.yaml` to make the metric optional.
+
+### Adding a new default metric or turning an existing optional metric into default
+
+Adding a new default metric is a breaking for a scraping receiver because it introduces an unexpected output for users 
+and additional load on metric backends. Steps to apply such change:
+
+1. If the metric doesn't exist yet, add one as an optional metric. Add a warning that the metric will be turned into 
+   default if the `enabled` option is not set explicitly to any value in user settings. A warning example: "WARNING: 
+   Metric `foo.bar` will be enabled by default in v0.65.0. If you don't want the metric to be emitted, please
+   disable it in the receiver settings."
+2. Remove the warning and update `metadata.yaml` to make the metric default.
+
+### Other changes
+
+Other breaking changes SHOULD follow similar strategies inspecting presence of `enabled` field in user settings. For 
```

Comment:
```suggestion
Other breaking changes SHOULD follow similar strategies that modify the `enabled` field in user settings. For 
```

Reply:
This correction changes the statement. We don't modify `enabled` field in the settings, we check if it's set only